README.md

```diff
@@ -9,16 +9,30 @@ A free and open source communications platform.
 ![Codecov](https://img.shields.io/codecov/c/gh/harmony-development/legato?style=for-the-badge)
 ![Mom Made Pizza Rolls](https://img.shields.io/badge/mom%20made-pizza%20rolls-green?style=for-the-badge)
 
-## Screenshots
-
-### App preview
-
-<img src="https://i.imgur.com/GHZN7BD.png" style="width:70%; text-align: center;">
-
-### Cool theming
-
-<img src="https://i.imgur.com/namHS1j.png" style="width:70%; text-align: center;">
-
-### Login screen
-
-<img src="https://i.imgur.com/6YmxSDO.png" style="width:70%; text-align: center;">
+## Building
+Required dependecies:
+- go
+- libvips
+- libvips-dev
+
+Then run:
+```
+go get -v -t -d ./...
+go build
+```
+
+## Usage
+To run server make sure to install:
+- go
+- postgres
+
+Make the following preparations:
+- Run command ```./legato -genkey``` to generate new server config
+- setup postgres user
+- update json config
+
+After all preparations run server:  
+```./legato```
```

Comment:
say to run ./legato once to generate the config in the beginning, then run it once it's all configured